debug/app/bespoke/components/customComponent.js

```diff
@@ -0,0 +1,27 @@
+SMK.HANDLER.set('BespokeTool--101', 'activated', (smk, tool) => {
+    tool.component = Vue.component('Zoom', {
+        data: function() {
+            return {
+                buttonText: "Fly to Victoria"
+            }
+        },
+        methods: {
+            handleZoom() {
+                const leafletMap = smk.$viewer.map;

```

Comment:
Maybe a comment stating that the smk object is a global that gives you access to everything within the smk app?